### PR TITLE
Feature sum

### PR DESCRIPTION
**Pull request by mreyes819**

### CODE REVIEW

File: client/js/table-view.js [hunk 1]
```diff
@@ -158,10 +158,10 @@ class TableView {
...
     const row = this.currentCellLocation.row
 
+    // checks to see if in col header when user clicks add row
+    // adds column to the right of the current column
+    // I should have implemented in table-model
     if (row === -1){
-      // echo, !DRY
-      // shifts column
-      // would like to put this funtion into table-model "shiftColumn"
       for(var c = this.model.numCols - 1; c > col; c--) {
```
Comment (speschellk) [inline client/js/table-view.js]: Replace "var" with either "const" or "let" throughout. Use "let" instead of "var" in for loops.

File: client/js/table-view.js [hunk 2]
```diff
@@ -187,10 +187,10 @@ class TableView {
...
+    // checks to see if in row header when user clicks add row
+    // adds row below the current row selected.
+    // I should have implemented in table-model
     if (col === 0){
-      // echo, !DRY
-      // shifts row
-      // would like to put this funtion into table-model "shiftRow"
       for(var r = this.model.numRows - 1; r > row; r--) {
         const rowValues = this.model.getRow(r);
         console.log(rowValues)
```
Comment (speschellk) [inline client/js/table-view.js]: Remove console logs you're no longer using.

File: public/js/bundle.js [hunk 3]
```diff
@@ -72,6 +72,7 @@ class TableModel {
   }
```
Comment (speschellk) [inline public/js/bundle.js:39]: Your bundle.js file should not be checked into git or GitHub, since it's a big ugly mess of a file that production environments (Heroku) will generate for you anyway. This is why we put it in the .gitignore file at the beginning of development.

Delete from GitHub and don't commit any modifications to bundle.js in git.

File: client/js/test/table-view.test.js [hunk 4]
```diff
@@ -20,7 +20,7 @@ describe('table-view', () => {
```
Comment (speschellk) [inline client/js/test/table-view.test.js:20]: Add a few more tests that test the functionality of your new feature. Especially test against non-happy paths and edge cases. For instance, “it correctly generates sum when the column includes non-number inputs”.

**Review by speschellk — CHANGES_REQUESTED** [review]

**Review by speschellk — APPROVED** [review]
